### PR TITLE
Add Cargo concordium

### PR DESCRIPTION
**Pull request by limemloh**
## Purpose

Add cargo-concordium from https://github.com/Concordium/concordium-wasm-smart-contracts/pull/59

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,22 @@
+.stack-work/
```
Comment (abizjak) [inline .gitignore:1]: We probably don't need that.

File: .gitmodules [hunk 2]
```diff
@@ -0,0 +1,3 @@
+[submodule "concordium-base"]
+	path = concordium-base
+	url = git@github.com:Concordium/concordium-base.git
```
Comment (abizjak) [inline .gitmodules:3]:
```suggestion
	url = ../concordium-base.git
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+This repository contains tools to support smart contracts on and off-chain.
+
+- [cargo-concordium](./cargo-concordium) which is a small tool for developing smart contracts. It uses the API exposed in `wasm-chain-integration` to execute smart contracts directly and can initialize and update smart contracts, in a desired state. See the `--help` option of the tool for details on how to invoke it.
+   It can also be used to build contracts embedded with schemas (see section about [contract schemas](#contract-schema)).
+
+
+# Contributing
+
+[![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.0-4baaaa.svg)](https://github.com/Concordium/.github/blob/main/.github/CODE_OF_CONDUCT.md)
```
Comment (abizjak) [inline README.md:12]: Usually we put the covenant at the top, don't we?

File: cargo-concordium/linux-cargo-concordium.Jenkinsfile [hunk 4]
```diff
@@ -0,0 +1,73 @@
+pipeline {
```
Comment (abizjak) [inline cargo-concordium/linux-cargo-concordium.Jenkinsfile:1]: Please put all these Jenkinsfiles into a `scripts` subfolder.

File: cargo-concordium/macos-cargo-concordium.Jenkinsfile [hunk 5]
```diff
@@ -0,0 +1,57 @@
+pipeline {
```
Comment (abizjak) [inline cargo-concordium/macos-cargo-concordium.Jenkinsfile:1]: And this one.

File: cargo-concordium/windows-cargo-concordium.Jenkinsfile [hunk 6]
```diff
@@ -0,0 +1,37 @@
+pipeline {
```
Comment (abizjak) [inline cargo-concordium/windows-cargo-concordium.Jenkinsfile:1]: And this one.

**Review by abizjak — APPROVED** [review]
Looks good, but please do the following

- [x] Move Jenkinsfiles into a scripts subfolder of cargo-concordium
- [ ] Correct the submodule path to be relative, this is what we use in other repositories.

File: README.md (hunk 3 above)
Comment (limemloh) [reply README.md]: I have move the badge a bit up

**Review by abizjak — APPROVED** [review]

**Pull request merged**